src/ChangelogGenerator/ChangelogConfig.php

```diff
@@ -28,8 +28,14 @@ class ChangelogConfig
     /** @var bool */
     private $includeOpen;
 
+    /** @var bool */
+    private $includeDate = false;
+
     /** @var mixed[] */
-    private $options = ['rootGitHubUrl' => self::DEFAULT_ROOT_GITHUB_URL];
+    private $options = [
+        'rootGitHubUrl' => self::DEFAULT_ROOT_GITHUB_URL,
+        'date_format' => 'Y-m-d',
```

Comment:
Use camel case like the other option. e.g. `dateFormat` instead of `date_format`